openlibrary/core/models.py

```diff
@@ -494,9 +494,9 @@ def get_users_rating(self, username: str) -> int | None:
         rating = Ratings.get_users_rating_for_work(username, work_id)
         return rating
 
-    def get_patrons_who_also_read(self):
+    def get_patrons_who_also_read(self, limit: int = 15):
         key = self.key.split('/')[-1][2:-1]
-        return Bookshelves.patrons_who_also_read(key)
+        return Bookshelves.patrons_who_also_read(key, limit)
```

Comment:
I wonder if this is cached or if we want it cached so we don't make lots of requests to the db.